File: Home2WorkAPI/src/main/java/it/gruppoinfor/home2workapi/model/Location.java
package it.gruppoinfor.home2workapi.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.io.Serializable;
import java.util.Date;


public class Location implements Serializable {

    @SerializedName("latlng")
    @Expose
    private LatLng latLng;
    @SerializedName("timestamp")
    @Expose
    private Date date;

    public LatLng getLatLng() {
        return latLng;
    }

    public void setLatLng(LatLng latLng) {
        this.latLng = latLng;
    }

    public Date getDate() {
        return date;
    }

    public void setDate(Date date) {
        this.date = date;
    }
}

File: Home2WorkAPI/src/main/java/it/gruppoinfor/home2workapi/location/Location.kt
package it.gruppoinfor.home2workapi.location

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName
import it.gruppoinfor.home2workapi.LatLng
import java.io.Serializable
import java.util.*


class Location : Serializable {

    @SerializedName("latlng")
    @Expose
    var latLng: LatLng = LatLng()
    @SerializedName("timestamp")
    @Expose
    var date: Date = Date()
}